packages/flutter/lib/src/widgets/fade_in_image.dart

```diff
@@ -301,6 +310,44 @@ class FadeInImage extends StatefulWidget {
   /// If not value set, it will fallback to [fit].
   final BoxFit? placeholderFit;
 
+  /// The rendering quality of the image.
+  ///
+  /// If the image is of a high quality and its pixels are perfectly aligned
+  /// with the physical screen pixels, extra quality enhancement may not be
+  /// necessary. If so, then [FilterQuality.none] would be the most efficient.
+  ///
+  /// If the pixels are not perfectly aligned with the screen pixels, or if the
+  /// image itself is of a low quality, [FilterQuality.none] may produce
+  /// undesirable artifacts. Consider using other [FilterQuality] values to
+  /// improve the rendered image quality in this case. Pixels may be misaligned
+  /// with the screen pixels as a result of transforms or scaling.
+  ///
+  /// See also:
+  ///
+  ///  * [FilterQuality], the enum containing all possible filter quality
+  ///    options.
+  final FilterQuality? filterQuality;
+
+  /// The rendering quality of the placeholder image.
+  ///
+  /// If the placeholder image is of a high quality and its pixels are perfectly aligned
```

Comment:
looks like some of these docs are repeated from above. Can you use a macro/template?

Reply:
should i put the docs for filterquality from the original file inside a template or just the copied one from this file? haven't worked with that feature yet so i don't know if i can reuse a template from another file or if that is a good practice in general. (https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/image.dart)

Reply:
You can use it across files. I would put the template in `image.dart` and re-use it here.

Reply:
If the [name](https://github.com/flutter/flutter/pull/110096/commits/0f17d9079ac2ab6587aace7910267512ec03055d) of the template conforms to the naming convention and it is okay that the placeholderFilterQuality variable [uses the exact same docs](https://github.com/flutter/flutter/pull/110096/commits/856c4c29255c39052e8a8a10cd8dda121b18639c), I believe this PR should be done.

Should I maybe remove the first paragraph out of the template so that i can replace "image" with "placeholder image" for the placeholderFilterQuality variable:
> The rendering quality of the image.

Reply:
> Should I maybe remove the first paragraph out of the template so that i can replace "image" with "placeholder image" for the placeholderFilterQuality variable:

That sounds like a good idea.

Reply:
> > Should I maybe remove the first paragraph out of the template so that i can replace "image" with "placeholder image" for the placeholderFilterQuality variable:
> 
> That sounds like a good idea.

https://github.com/flutter/flutter/pull/110096/commits/6ad3831b56559ed6ceb19b56fff31ecb52350bf1 https://github.com/flutter/flutter/pull/110096/commits/b4216287beb11ce5db73edb863eec6947fccd5cc
done